builds/e2e/templates/connectivity-deploy.yaml

```diff
@@ -73,6 +77,14 @@ steps:
           -edgeletBranchName "${{ parameters['edgelet.branchName'] }}" \
           -hostPlatform "${{ parameters['metricsCollector.hostPlatform'] }}" \
           -storageAccountConnectionString "${{ parameters['testResultCoordinator.storageAccountConnectionString'] }}" \
+          -devOpsAccessToken $(System.AccessToken) \
```

Comment:
Need these 2 parameter to query rest api for build cancel

Reply:
whose accessToken are you using? Can we have a generic one for the team use?

Reply:
This is a security token used by the running build. https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#systemaccesstoken